doc/howto/storage_backup.md

```diff
@@ -0,0 +1,109 @@
+# How to back up storage volumes
+
+There are two ways to back up your storage volumes: with snapshots or with export files.
+Snapshots are cheap and easy to create, while export files are more reliable and can also be used to restore a volume into a different storage pool.
+
+```{note}
+Storage volumes might be attached to an instance, but they are not part of the instance.
+Therefore, the content of a storage volume is not stored when you back up your instance.
+You must back up the data of your storage volume separately.
+```
+
+## Use snapshots for backup
+
+A snapshot saves the state of the storage volume at a specific time.
+It is stored in the same storage pool as the volume itself and cannot be used outside of the storage volume.
```

Comment:
I think it is possible to copy (so effectively restore) a snapshot to a new volume on a different pool.

Reply:
Would that be using the alternative syntax again? (So something like `lxc storage volume copy <source_pool_name>/<source_volume_name>/<source_snapshot_name> <target_pool_name>/<target_volume_name>`?)

Reply:
Yes thats right, one can also specify a remote target too.